SU2_CFD/src/solvers/CNEMOEulerSolver.cpp

```diff
@@ -1586,7 +1583,7 @@ void CNEMOEulerSolver::SetNondimensionalization(CConfig *config, unsigned short
 
     switch(config->GetKind_FluidModel()){
     case USER_DEFINED_NONEQ:
-      ModelTable << "User defined";
+      ModelTable << "SU2 Two-Temp";
```

Comment:
hi @WallyMaier  actually I don't really agree here, that's why I had changed it in the past. 2T model is the whole NEMO solver model, despite of the fluidmodel. fluid model is either mutation++ library or the hardcoded (user defined), and both use 2T model. so this is not very consistent. I suggest we change back ?

Reply:
I was debating this as well.  I feel like "User Defined" wasnt descriptive enough.    

Would be "SU2 NonEq" be better?   If not, Ill change it back

Reply:
well not really, cause that is the same problem, both fluid models are NonEq. The only thing that distinguishes them is really that one is hardcoded and the other one is the coupling to mutation++...

Reply:
Okie dok.  How about?
SU2 NonEq
Mutation++ NonEq

Reply:
for me is ok, yes